api-client/src/models/__tests__/LocationModel.spec.ts

```diff
@@ -124,3 +124,50 @@ describe('convertToPoint', () => {
     expect(location.convertToPoint(path, thumbnail, urlSlug)).toBeNull()
   })
 })
+describe('get coordinates', () => {
```

Comment:
you should put those tests in the location model scope.

Reply:
Also the test under  describe('location', ?